app/components/ui/persistent-collapsable-container/component.js

```diff
@@ -1,19 +1,23 @@
 import Ember from "ember";
+import computed from "ember-computed-decorators";
 
 export default Ember.Component.extend({
   preferencesService: Ember.inject.service(),
 
-  isClosedDynamicComputed: function() {
-          return Ember.Object.extend({
-               value: Ember.computed.alias(`preferencesService.preferencesData.${this.get("settingsKey")}`)
-          }).create({preferencesService: this.get("preferencesService")});
-    }.property("preferencesService", "settingsKey"),
+  @computed("preferencesService.preferencesData", "settingsKey")
```

Comment:
Unfortunately, I don't think this will work since there is no dependent key set to trigger the recompute when the value toggles.

I think you should write an integration test for the `persistent-collapsable`. Toggling `preferencesService.setting` should result in the container css updated.

What I meant with my earlier comment was we could rewrite as:

```
isClosedDynamicComputed: Ember.computed("preferencesService", "settingsKey", function() {
  return Ember.Object.extend({
    value: Ember.computed.alias(`preferencesService.preferencesData.${this.get("settingsKey")}`)
  }).create({preferencesService: this.get("preferencesService")});
})
```

Instead of using `property`